safehttp/plugins/hsts/hsts.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hsts
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/google/go-safeweb/safehttp"
+)
+
+// Plugin implements automatic HSTS functionality.
+type Plugin struct {
+	maxAge            uint64
+	includeSubDomains bool
+	preload           bool
+}
+
+// NewPlugin creates a new HSTS plugin with safe defaults.
+func NewPlugin() Plugin {
+	return Plugin{
+		maxAge:            63072000, // Two years in seconds.
```

Comment:
No need to capitalise the comment, I think

Reply:
I don't see how having `// two years in seconds.` is any better. I'll keep it as `// Two years in seconds.` to respect capitalization in beginning of sentences.

Reply:
I was looking at this: https://golang.org/src/go/doc/example.go and it seems like the standard is no capitalisation or full stop for inline comments :) It's not a big thing anyway

Reply:
ok, then I will change it :)